modules/ztp-sno-du-configuring-disk-encryption-with-pcr-protection.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-du-configuring-disk-encryption-with-pcr-protection_{context}"]
+= Enabling disk encryption with TPM and PCR protection
+
+You can use the `diskEncryption` field in the `SiteConfig` custom resource (CR) to configure disk encryption with Trusted Platform Module (TPM) and Platform Configuration Registers (PCRs) protection. 
+
+Configuring the `SiteConfig` CR enables disk encryption at the time of cluster installation.
+
+.Prerequisites
+
+* You have installed the {oc-first}.
+* You have logged in as a user with `cluster-admin` privileges.
+* You read the "About disk encryption with PCR protection" section.
```

Comment:
Suggestion: I think it would help the user experience a lot here if an _Additional resources_ section could be added after this module with a link to "About disk encryption with PCR protection" - I wasn't able to find this section on the page, so I'm not sure where it's located in the docs.

Reply:
Updated as per the suggestion.